src/guide/receive_onchain.md

```diff
@@ -250,7 +276,15 @@ except Exception as error:
 <section>
 
 ```go
-if swapInfo, err := sdkServices.ReceiveOnchain(); err != nil {
+// Optional user-selected dynamic fees
+lspInfo, err := sdkServices.LspInfo()
```

Comment:
```
id, err := sdkServices.LspId()
lspInfo, err := sdkServices.FetchLspInfo(id)
```

Reply:
Preparing PR to expose `lsp_info()` via the bindings.